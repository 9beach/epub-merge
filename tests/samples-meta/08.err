[90m작가: Bad
Author
; Really Really
Bad Author [But has
Lovely Name] -> 배수아[0m
[90m번역: <empty> -> Deborah Smith[0m
[90m도서번호: BAD
ISBN XXXXX -> ISBN 978-89-9470-250-6 13191[0m
