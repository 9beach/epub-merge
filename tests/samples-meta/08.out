제목: 뱀과 물
작가: 배수아
번역: Deborah Smith
출판일: 2017-12-07
언어: ko
도서번호: ISBN 978-89-9470-250-6 13191
